Library/Homebrew/test/support/fixtures/cask/Casks/json-loadable-cask.rb

```diff
@@ -0,0 +1,39 @@
+# Used to test JSON generation with a cask that only uses
+# stanzas that can be loaded from JSON.
+cask "json-loadable-cask" do
+  version "1.2.3"
+  sha256 "67cdb8a02803ef37fdbf7e0be205863172e41a561ca446cd84f0d7ab35a99d94"
+
+  url "https://cachefly.example.app/releases/Example_#{version}.zip"
+  name "Example"
+  desc "Little bit of everything"
+  homepage "https://www.example.app/"
+
+  auto_updates true
+  conflicts_with formula: "nothing"
+  depends_on cask: "something"
+  container type: :naked
+
+  app "Example.app"
+  installer script: {
+    executable:   "~/just/another/path/install.sh",
```

Comment:
I would have liked to use `staged_path` here with `Cask::Cask.generating_hash!` to test out path subs but it ended up being flaky. I don't think we can have multiple tests with that logic right now otherwise they might intermittently fail when run in parallel. Kind of annoying.